scripts/bolt/internal/slurm_merge.sh

```
@@ -0,0 +1,175 @@
+     --container-mounts="$MOUNTS" \
+     --container-workdir=/workspace \
+     --no-container-mount-home \
+     bash -lc '
+        set -euo pipefail
+        export PATH=/builds/llvm/bin:$PATH
+        export BOLT_WORK_DIR='"$RUNDIR"'/work
+
+        # 1) Reconstruct the ORIGINAL libs: install the wheel + back up ELFs.
+        #    No instrument -- the collect jobs already produced the .fdata.
+        _extract='"$RUNDIR"'/extract
+        mkdir -p "$_extract" && tar xzf /builds/'"$TARBALL_NAME"' -C "$_extract" 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)